### PR TITLE
Link to system admin from realm select

### PR DESCRIPTION
**Pull request by whaught**
## Proposed Changes

* Link directly to system-admin page from realm-select
   * Note that if the admin is not a member of any realms, we go straight to admin already. https://github.com/google/exposure-notifications-verification-server/blob/f76a3cea09d53d076a47a204ce9c948e183c8f35/pkg/controller/login/select.go#L50

* Remove "launch admin" from navigation if already there

straighttoadmin

### CODE REVIEW

File: cmd/server/assets/header.html [hunk 1]
```diff
@@ -103,10 +103,12 @@ <h6 class="dropdown-header">Manage realm</h6>
       {{end}}
 
       {{if .currentUser.SystemAdmin}}
+      {{if not (.currentPath.IsDir "/admin")}}
```
Comment (mikehelmick) [inline cmd/server/assets/header.html:106]: I don't see any reason to remove this - we don't do it w/ other pages

Comment (whaught) [reply cmd/server/assets/header.html:106]: maybe "Launch" just feels funny to me when you're already there

Comment (mikehelmick) [reply cmd/server/assets/header.html:106]: what if we link to specific pages here instead?

image

Comment (whaught) [reply cmd/server/assets/header.html]: How about we just highlight the active nav item (see most recent commit)

**Review by mikehelmick — APPROVED** [review]

**Pull request merged**